templates/site/js/settings.js

```diff
@@ -19,60 +14,19 @@ document.addEventListener("DOMContentLoaded", function () {
 
 function setUserDetails(){
     //TODO Remove placeholder username once design is approved
-    var placeholder_username = "XYZ";
+    const placeholder_username = "XYZ";
```

Comment:
can this be removed now?

Reply:
Yes, this should be fine even without Shruti's final review.

Removed the placeholder.